app/models/pglogical_subscription.rb

```diff
@@ -229,8 +281,7 @@ def update_subscription
   # sets this instance's password field to the one in the subscription dsn in the database
   def find_password
     return password if password.present?
-    s = subscription_attributes.symbolize_keys
-    dsn_hash = PG::DSNParser.parse(s.delete(:subscription_dsn))
+    dsn_hash = PG::DSNParser.parse(subscription_dsn)
```

Comment:
I added it in an attempt to be efficient, since otherwise the `find_password` method is making an extra call to the `subscriptions` method to get information it had but deleted earlier.

Reply:
But this leads to us passing the entire dsn string to the UI as an attribute of this object, right? We can't have that. This is exactly why we go through such trouble to hide the password.

Reply:
Ok, understood, but that is precisely the sort of thing that needs commenting. Without being intimately familiar with pglogical, that is non-obvious.